indico/modules/events/papers/__init__.py

```diff
@@ -44,7 +44,7 @@ def _extend_editing_menu(sender, event, **kwargs):
     if (has_contributions_with_user_paper_submission_rights(event, session.user) or
             event.cfp.is_staff(session.user)):
         return SideMenuItem('papers', _('Call for Papers'), url_for('papers.call_for_papers', event),
-                            section='organization')
+                            section='workflows')
```

Comment:
Same here, most likely the section can be removed from all menu items that are returned for `event-editing-sidemenu`